src/components/Navigation/Pages/PageBrowse/PageBrowse.js

```diff
@@ -156,7 +156,6 @@ class PageBrowse extends SystemManagedList {
       <ButtonsBar
         hasChanges={!this.hasChanges()}
         revertAll={() => this.revertAll('browse')}
-        toggleCollapseExpandAll={this.toggleCollapseExpandAll}
         doPromptContribute={this.doPromptContribute}
```

Comment:
toggleCollapseExpandAll can now be undefined in the props.

Reply:
toggleCollapseExpandAll involves list update of complexity n2, and can potentially be expensive.  See SystemManagedList.toggleCollapseExpandAll, PageBrowse.updateList, and update in listReducer. That is why it is currently set to be hidden here.